build/speedrun_web.py

```diff
@@ -0,0 +1,475 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#   "marimo",
+#   "numpy==2.2.6",
+#   "pandas==2.3.0",
+#   "plotly==6.2.0",
+#   "pyarrow",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.14.10"
+app = marimo.App(
+    width="full",
+    css_file="custom.css",
+    html_head_file="head.html",
+)
+
+
+@app.cell
+def _():
+    import marimo as mo
+
+    return (mo,)
+
+
+@app.cell
+def _(mo):
+    mo.Html(
+        f"""
+    <header class="bg-marin-dark text-white shadow-lg">
```

Comment:
For actual web developers who would rather just write React...what should they do here?

Reply:
Marimo supports HTML or Markdown embedding, but (despite using React under the hood) I don't think you can natively embed react.